tests/networkpolicy_test.go

```diff
@@ -1,173 +1,264 @@
-		})
+			It("[test_id:1516] should be failed to reach serverVMI from clientVMI", func() {
+				By("Connect serverVMI from clientVMI")
+				assertPingFail(clientVMI, serverVMI)
+			})
 
-		It("[test_id:1517] should be successful to reach vmib from vmic", func() {
-			By("Connect vmib from vmic")
-			assertPingSucceed(vmic, vmib)
+			It("[test_id:1517] should be successful to reach clientVMI from clientVMIAlternativeNamespace", func() {
+				By("Connect clientVMI from clientVMIAlternativeNamespace")
+				assertPingSucceed(clientVMIAlternativeNamespace, clientVMI)
+			})
+
+		})
+		Context("and an allow all http ports policy is populated", func() {
+			var policy *networkv1.NetworkPolicy
+			BeforeEach(func() {
+				port80 := intstr.FromInt(80)
+				port81 := intstr.FromInt(81)
+				tcp := corev1.ProtocolTCP
+				policy = &networkv1.NetworkPolicy{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: serverVMI.Namespace,
+						Name:      "allow-all-http-ports",
+					},
+					Spec: networkv1.NetworkPolicySpec{
+						PodSelector: metav1.LabelSelector{},
+						Ingress: []networkv1.NetworkPolicyIngressRule{
+							{
+								Ports: []networkv1.NetworkPolicyPort{
+									{Port: &port80, Protocol: &tcp},
+									{Port: &port81, Protocol: &tcp},
+								},
+							},
+						},
+					},
+				}
+				createNetworkPolicy(policy)
+			})
+			AfterEach(func() {
+				waitForNetworkPolicyDeletion(policy)
+			})
+			It("[test_id:2774] should allow http traffic for ports 80 and 81 from clientVMI to serverVMI", func() {
+				PingHTTPEventually(clientVMI, serverVMI, 80).Should(Succeed(), "should allow HTTP traffic from clientVMI go serverVMI with port 80")
+				PingHTTPEventually(clientVMI, serverVMI, 81).Should(Succeed(), "should allow HTTP traffic from clientVMI go serverVMI with port 81")
+			})
+		})
+		Context("and an allow only 80 http port policy is populated", func() {
+			var policy *networkv1.NetworkPolicy
+			BeforeEach(func() {
+				port80 := intstr.FromInt(80)
+				tcp := corev1.ProtocolTCP
+				policy = &networkv1.NetworkPolicy{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: serverVMI.Namespace,
+						Name:      "allow-http80-port",
+					},
+					Spec: networkv1.NetworkPolicySpec{
+						PodSelector: metav1.LabelSelector{},
+						Ingress: []networkv1.NetworkPolicyIngressRule{
+							{
+								Ports: []networkv1.NetworkPolicyPort{
+									{Port: &port80, Protocol: &tcp},
+								},
+							},
+						},
+					},
+				}
+				createNetworkPolicy(policy)
+			})
+			AfterEach(func() {
+				waitForNetworkPolicyDeletion(policy)
+			})
+			It("[test_id:2775] should allow http traffic at port 80 and deny at port 81 from clientVMI to serverVMI", func() {
+				PingHTTPEventually(clientVMI, serverVMI, 80).Should(Succeed(), "should allow HTTP traffic from clientVMI go serverVMI with port 80")
+				PingHTTPEventually(clientVMI, serverVMI, 81).ShouldNot(Succeed(), "should deny HTTP traffic from clientVMI go serverVMI with port 81")
```

Comment:
Same here; a check from the VMI on the other namespace is valuable, imo.

Reply:
Done

Reply:
After talking we decided not to do it.